backend/app.py

```diff
@@ -54,4 +57,4 @@ def configure_app(self):
     def serve(self):
         import waitress
         waitress.serve(self.app, host=self.host, port=self.port)
-        return
\ No newline at end of file
+        return
```

Comment:
```suggestion
```

nit: If you aren't returning leave it out

Reply:
Noticed there is a lot more return nothings in these files (that weren't affected by this PR. So please remove those as well.

Reply:
Done.